### PR TITLE
Upgrade to 4.41

### PR DESCRIPTION
**Pull request by SameerK-MSFT**

### CODE REVIEW

**Review by gladjohn — APPROVED** [review]
looks good @SameerK-MSFT

File: 2-With-broker/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs [hunk 1]
```diff
@@ -24,39 +24,42 @@ private void OnSignInSignOut(object sender, EventArgs e)
...
+                    {
+                        case Device.Android:
+                            builder = builder.WithRedirectUri(App.BrokerRedirectUriOnAndroid);
+                            break;
+                        case Device.iOS:
+                            builder = builder.WithIosKeychainSecurityGroup("com.microsoft.adalcache");
+                            builder = builder.WithRedirectUri(App.BrokerRedirectUriOnIos);
+                            break;
+                        case Device.UWP:
+                            builder = builder.WithExperimentalFeatures();
```
Comment (bgavrilMS) [inline 2-With-broker/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs:44]: no longer needed.

**Review by bgavrilMS — APPROVED** [review]

File: 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj [hunk 2]
```diff
@@ -42,6 +42,7 @@
     <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
     <BundleAssemblies>false</BundleAssemblies>
     <AndroidEnableMultiDex>true</AndroidEnableMultiDex>
+    <AndroidSupportedAbis>arm64-v8a;x86;armeabi-v7a</AndroidSupportedAbis>
```
Comment (pmaytak) [inline 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj:45]: Is this needed?

**Review by pmaytak — APPROVED** [review]

**Review by neha-bhargava — APPROVED** [review]
👍

File: 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj (hunk 2 above)
Comment (SameerK-MSFT) [reply 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj:45]: This is to test on arm phones. Else developer will have to modify it.

File: 2-With-broker/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs (hunk 1 above)
Comment (SameerK-MSFT) [reply 2-With-broker/UserDetailsClient/UserDetailsClient/MainPage.xaml.cs]: Removed.

**Pull request merged**